android/modules/analytics/src/java/ti/modules/titanium/analytics/AnalyticsModule.java

```diff
@@ -6,204 +6,62 @@
  */
 package ti.modules.titanium.analytics;
 
-import java.util.HashMap;
-import java.util.Iterator;
+import com.appcelerator.aps.APSAnalytics;
```

Comment:
Can't we just nuke this file?

Reply:
Thought about that too but then it has to be a major update since it will break apps that use `setOptedOut` or other analytics mehtods. Yes they shouldn't use them but I think this change will be better for the first "non appc" release.

Reply:
Can we add a debug log message at either build or run time if someone is using `setOptedOut` that the feature is deprecated?

Reply:
you are right! Added a deprection debug log.

Reply:
Local test are fine too, just the Clipboard issue (https://github.com/tidev/titanium_mobile/pull/12368) from time to time

Reply:
Looks good, thanks!